pyqtorch/utils.py

```diff
@@ -125,6 +125,10 @@ def param_dict(keys: Sequence[str], values: Sequence[Tensor]) -> dict[str, Tenso
     return {key: val for key, val in zip(keys, values)}
 
 
+class Density_Matrix(Tensor):
```

Comment:
This is extremely minor but isn't it better to be DensityMatrix instead?

Reply:
I will change.